packages/camlidl/camlidl.1.07/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "Nicolas Berthier <m@nberth.space>"
+authors: ["Xavier Leroy"]
+homepage: "https://github.com/xavierleroy/camlidl"
+dev-repo: "git+https://github.com/xavierleroy/camlidl.git"
+bug-reports: "https://github.com/xavierleroy/camlidl/issues"
+license: "QPL-1.0 and LGPL-2 with exceptions"
+build: [
+  ["mv" "config/Makefile.unix" "config/Makefile"] {os != "win32"}
+  ["mv" "config/Makefile.mingw" "config/Makefile"] {os = "win32"}
+  [make "all"]
+]
+patches: [
+  "disable-fatal-warn-31.diff" {ocaml:version >= "4.03"}
+  "camlidl-camlidl106.patch" {os = "win32"}
```

Comment:
```suggestion
```
This patch doesn't exist

Reply:
Right, sorry. I forgot to remove it from this section.

Reply:
Fixed.